packages/pg-meta/test/run-tests.sh

```
@@ -0,0 +1,41 @@
+  fi
+  PG_TEST_PORT=$((PG_TEST_PORT + 1))
+done
+export PG_TEST_PORT
+
+DATABASE_URL="postgresql://postgres:postgres@localhost:${PG_TEST_PORT}"
+export DATABASE_URL
+
+cleanup() {
+  cd "$DB_DIR"
+  docker compose down 2>/dev/null || true
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)